### DISCUSSION TITLE
QuestPDF pdf file version

### DISCUSSION THREAD

**Discussion by redSkullBoy** [QuestPDF/QuestPDF#862] [Q&A] [answered]
Hi, is there a QuestPDF function to specify the pdf file version, I need the PDF 1.3 or PDF 1.4 version.

**Comment by deMD** [comment]
The PDF is, according to Acrobat reader, version 1.4 by default. I do wonder though if it is possible to generate a version 1.7+ version espacially since 1.7 is an ISO standard: https://en.wikipedia.org/wiki/History_of_PDF

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

QuestPDF does not expose a setting to choose the raw PDF version header (such as 1.3 or 1.4), and there is no way to target a lower version than the default output. What is configurable is standards conformance: through `PDFA_Conformance` and `PDFUA_Conformance` in [Document Settings](https://www.questpdf.com/concepts/document-settings.html) you can produce PDF/A (ISO 19005) and PDF/UA (ISO 14289) documents, with PDF/A-2 and PDF/A-3 being based on the ISO 32000-1 (PDF 1.7) standard.

Should you encounter any further problems, please don't hesitate to open a new issue.
Thank you!
